core/kernel/dt_driver.c

```diff
@@ -245,6 +245,8 @@ void *dt_driver_device_from_node_idx_prop(const char *prop_name,
 
 	prop = fdt_getprop(fdt, nodeoffset, prop_name, &len);
 	if (!prop) {
+		DMSG("No property %s found in node %s", prop_name,
```

Comment:
`Property %s not found in node %s`
or perhaps even
`Property %s missing in node %s`?

Reply:
ok. `Property %s missing in node %s`.